crates/top/re_sdk/src/lenses/ast.rs

```diff
@@ -196,13 +211,65 @@ impl Lens {
             ));
         }
 
+        let mut time_builders = ahash::HashMap::default();
+        for time in &self.time_columns {
+            let entity_path = time.new_entity_path.as_ref().unwrap_or(chunk.entity_path());
+
+            let time_entry = time_builders.entry(entity_path).or_insert_with(
+                <std::collections::HashMap<TimelineName, re_chunk::TimeColumn>>::default,
```

Comment:
nit: doesn't `or_insert_default` do the trick here? If it's a type deduction issue, I'd rather type spec `let mut time_builders` :)